doc/lightningd.8.txt

```diff
@@ -0,0 +1,185 @@
+  location.
+  Defaults to 'config'.
+*--lightning-dir*='DIR'::
+  Set the directory for the C-Lightning daemon.
+  Defaults to '$HOME/.lightning'.
+
+MORE OPTIONS
+------------
+Command line options are mirrored as configuration options in the
+configuration file, so 'foo' in the configuration file simply
+becomes '--foo' on the command line, and 'foo=bar' becomes
+'--foo=bar'.
+
+See lightningd-config(5) for a comprehensive list of all
+available options.
+
+LOGGING AND COMMANDING C-LIGHTNING
+-----------------------------
+By default, C-Lightning will log to the standard output.
+To log to a specific file, use '--log-file=PATH'.
+Sending SIGHUP will cause C-Lightning to reopen this file,
+for example to do log rotation.
+
+C-Lightning will set up a Unix domain socket for receiving
+commands.
+By default this will be the file 'lightning-rpc' in your
+specified 'lightning-dir'.
+You can use lightning-cli(1) to send commands to C-Lightning
+once 'lightningd' has started; you need to match the
+'--lightning-dir' and '--rpc-file' options between them.
+
+Commands for C-Lightning are described in various manpages
+in section 7, with the common prefix 'lightning-'.
+
+QUICK START
+-----------
+First, decide on and create a directory for 'lightning-dir',
+or just use the default '$HOME/.lightning'.
+Then create a 'config' file in this directory containing your
+configuration.
+
+Your other main preparation would be to set up a mainnet Bitcoin
+fullnode, i.e. run a bitcoind(1) instance.
+The rest of this quick start guide will assume you are reckless
+and want to spend real funds on Lightning.
+Indicate 'network=bitcoin' in your 'config' file explicitly.
+
+C-Lightning needs to communicate with the Bitcoin Core RPC.
+You can set this up using 'bitcoin-datadir', 'bitcoin-rpcconnect',
+'bitcoin-rpcport', 'bitcoin-rpcuser', and 'bitcoin-rpcpassword'
+options in your 'config' file.
+
+Finally, just to keep yourself sane, decide on a log file name
+and indicate it using 'log-file=lightningd.log' in your 'config' file.
+You might be interested in viewing it periodically as you follow
+along on this guide.
+
+Once the *bitcoind* instance is running, start lightningd(8):
+
+  $ lightningd --lightning-dir=$HOME/.lightning --daemon
+
+This starts *lightningd* in the background due to the '--daemon'
+option.
+
+Check if things are working:
+
+  $ lightning-cli --lightning-dir=%HOME/.lightning help
+  $ lightning-cli --lightning-dir=%HOME/.lightning getinfo
+
+The *getinfo* command in particular will return a 'blockheight'
+field, which indicates the block height to which *lightningd*
+has been synchronized to (this is separate from the block height
+that your *bitcoind* has been synchronized to, and will always
+lag behind *bitcoind*).
+You will have to wait until the 'blockheight' has reached the
+actual blockheight of the Bitcoin network.
+
+Before you can get funds offchain, you need to have some funds
```

Comment:
I wonder if we should recommend the `helpme` plugin at this point.  It does things like connect to nodes on the network for you, and suggest things.

But that might have to wait until it's more mature?

Reply:
If it does not come standard with a C-Lightning release, it is hard to recommend in the C-Lightning manpage.  I am already wary about suggesting `dig` and `https://1ml.com` but could not find an easier way to  get to *some* node, but at least those are common tools that can be easily acquired.

Reply:
Agreed.  Let's add a bootstrap script.